README.md

```diff
@@ -15,22 +15,22 @@ You can find the content description, speakers and trainers at:
 
 ## [👩🏽‍🏫 Core & Elective Courses](CORE_AND_ELECTIVE_COURSES)
 
-| Title  | Trainers  | Short Description  | Materials  | Recording |
+| Course | Title  | Trainers  | Short Description  | Materials  | Video Recording |
```

Comment:
This table does not render good anymore :(

<img width="885" alt="Screenshot 2019-09-04 at 16 16 21" src="https://user-images.githubusercontent.com/152863/64268066-5a7c0300-cf2f-11e9-83e1-b5f4374fb2a6.png">

Reply:
Thanks for the catch! I believe I've successfully fixed this!